### PR TITLE
Fix edge to edge mode

### LINKED ISSUE #648: When a file preview is opened, the delete button isn't displayed
It's appearing when we go to file info fragment, then come back to file menu

### PR DESCRIPTION
**Pull request by FabianDevel**
closes #648 and #651

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/ui/fileList/fileDetails/FileDetailsFragment.kt [hunk 1]
```diff
@@ -98,6 +100,7 @@ class FileDetailsFragment : FileDetailsSubFragment() {
                 WindowInsetsCompat.CONSUMED
             }
         }
+        super.onStart()
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/fileDetails/FileDetailsFragment.kt:103]: Put it in the beginning of the method.

File: app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt [hunk 2]
```diff
@@ -164,8 +164,11 @@ class PreviewSliderFragment : Fragment(), FileInfoActionsView.OnItemClickListene
         }
 
         configureBottomSheetFileInfo()
+    }
 
+    override fun onStart() {
         setupTransparentStatusBar()
+        super.onStart()
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt:171]: Put it in the beginning of the method.

File: app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt [hunk 3]
```diff
@@ -282,6 +285,7 @@ class PreviewSliderFragment : Fragment(), FileInfoActionsView.OnItemClickListene
                     header?.setMargin(top = top, right = right, left = left)
                     bottomSheetBehavior.peekHeight = getDefaultPeekHeight() + bottom
                     bottomSheetBehavior.expandedOffset = top
+                    bottomSheetFileInfos.setPadding(0, 0, 0, top + bottom)
```
Comment (sirambd) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt:288]: Need comment here

**Review by sirambd — CHANGES_REQUESTED** [review]
Waiting for feedback resolution

File: app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt (hunk 3 above)
Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewSliderFragment.kt:290]: done :)

**Pull request merged**